server/metadata/namespaces.go

```diff
@@ -27,6 +28,7 @@ import (
 
 const (
 	NamespaceSubspaceName = "namespace"
+	DatabaseKey           = "db"
```

Comment:
Can you also add `namespace`, `db` to the reserved/restrictions list.  Also, this key is already defined in `dictionary.go`, better to use that.

Reply:
I made some validation. please let me know if there is any place else I missed it.